File: 2114. Maximum Number of Words Found in Sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int max = 0;
        for(int i = 0; i<sentences.length; i++){
            String[] temp = sentences[i].split(" ");
            if(temp.length > max) max = temp.length;
        }
        return max;
    }
}

File: 2114. Maximum Number of Words Found in Sentences.ts
function mostWordsFound(sentences: string[]): number {
    let res: number = 0;
    sentences.forEach(function (str){
        let max = 1;
        Array.from(str).forEach(function (char){
           if(char === ' '){
               max++;
           } 
        })
        res = Math.max(res, max);
    })
    
    return res;
};
